src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,22 @@
+@import "../utils/variables.scss";
+
+.catalog {
+  margin: 50px auto 0;
```

Comment:
You can't use margin on BEM blocks, use mix of BEM block and element for this purpose